thirtyinch/src/test/java/net/grandcentrix/thirtyinch/TiLifecycleObserverTest.java

```diff
@@ -303,14 +302,14 @@ public void onChange(final TiPresenter.State state,
         mPresenter.detachView();
 
         final Object[] beforeLast = states.get(states.size() - 2);
-        assertEquals(beforeLast[0], TiPresenter.State.VIEW_DETACHED);
-        assertEquals(beforeLast[1], false);
-        assertNotNull(beforeLast[2]);
+        assertThat(beforeLast[0]).isEqualTo(TiPresenter.State.VIEW_DETACHED);
+        assertThat(beforeLast[1]).isEqualTo(false);
```

Comment:
better use `.isFalse()`

Reply:
In these cases, it's AbstractObjectAssert, which does not have isTrue()/isFalse().